### ISSUE TITLE
the loss plot for running the program on a given data set

### ISSUE THREAD

**Issue by wenouyang** [orobix/retina-unet#14]
Hi,

I applied your model to another data set. I let the program run 60 epochs, and the loss plot is shown as following.  In accordance with the plot, do you think the program has converged? It seems to me that loss change has become flat, but the change of loss is not very significant between epoch 1 and epoch 60. 

Any insightful suggestions are very appreciated!

![capture](https://cloud.githubusercontent.com/assets/5430158/21578414/7eb68454-cf44-11e6-817d-0272449b13c1.JPG)

**Comment by dcorti** [comment]
Hi

It looks like the program converged, but indeed the loss function did not decrease significantly. Usually, with the retina DRIVE dataset, the training loss at least halves before flattening. 
Probably in your case the program did not learn much, what kind of dataset did you try?

**Comment by wenouyang** [comment]
Hi, the data set is remote sensing data for field survey.  Currently, I have 125 pairs of raw training data with size 512*512.  I heavily sampled from this training set to have 100K training set with 64*64 patches. 

Will this kind of sampling process cause any potential issue due to the heavy overlapping among sampled training set?

**Comment by dcorti** [comment]
I'm not sure, since I did strong ovelap as well but it did not create problems. It's hard to guess where the problem could be.
